tests/test_chromatic.py

```diff
@@ -1717,6 +1717,31 @@ def test_ChromaticAiry():
     t2 = time.time()
     print 'time for %s = %.2f'%(funcname(),t2-t1)
 
+def test_chromatic_fiducial_wavelength():
+    """ Check that chromatic code can handle profiles with flux(effective_wavelength) = 0.
+    """
+    import time
+    t1 = time.time()
+
+    waves = np.arange(500., 600.1, 10.)
+    blue_flux = waves < 550.0
+    red_flux = waves > 550.0
+    bp = galsim.Bandpass(galsim.LookupTable(waves, waves**0, interpolant='linear'))
+    blue_sed = galsim.SED(galsim.LookupTable(waves, blue_flux, interpolant='linear'))
+    red_sed = galsim.SED(galsim.LookupTable(waves, red_flux, interpolant='linear'))
+
+    gal1 = galsim.Gaussian(fwhm=1) * blue_sed
+    gal2 = galsim.Gaussian(fwhm=1) * red_sed
+    img1 = gal1.drawImage(bp)
+    img2 = gal2.drawImage(bp)
+    # Didn't see an obvious np.testing method for checking finiteness, so just use assert.
```

Comment:
I think you're right that there isn't one, so if you want to use a np.testing routine it would have to be something like this:
```
np.testing.assert_equal(np.isfinite(img1.array.sum()).astype(int), 1, "whatever")
```